src/audio.c

```diff
@@ -1203,7 +1204,7 @@ static int tx_thread(void *arg)
 	uint64_t ts = 0;
 
 	mtx_lock(tx->mtx);
-	while (tx->thr.run) {
+	while (re_atomic_weak(&tx->thr.run)) {
```

Comment:
Maybe we can replace this one with stronger memory ordering and can remove some mutex locks.